crates/cargo-contract/src/main.rs

```diff
@@ -121,6 +121,9 @@ enum Command {
     /// Call a contract
     #[clap(name = "call")]
     Call(CallCommand),
+    /// Executes a call
+    #[clap(name = "exec")]
+    Exec(CallCommand),
```

Comment:
I think this would be confusing. Either we use the `--exec` option or have an `exec` command, but not both.

Reply:
I realise this is what is specified in https://github.com/paritytech/cargo-contract/issues/979

Reply:
So what's the consensus here? Do we want to have both or keep just one?

Reply:
At the moment each command maps directly to its corresponding dispatchable in `pallet-contracts`: `upload`, `remove` `instantiate`, `call`. And each of those (except `remove`) can be run via RPC with no effect ("dry run"), or run as a transaction ("executed"). So having a flag makes sense for running each command in a particular mode for this mental model.

So having a new `exec` command as a shortcut doesn't fit with this model. It may well be nice to have, but we have to ask is having to type `call --exec` instead of just `exec` that much of an improvement, to justify the introduction of a new command and new terminology?

I am open to having `exec` and/or `query` commands, with good justification. We could do these as simple follow up PRs

Reply:
Agreed. It's not a big deal. I'll remove `exec` command then